### DISCUSSION TITLE
any way to remote update wz_mini?

### DISCUSSION THREAD

**Discussion by idaband** [gtxaspec/wz_mini_hacks#141] [Q&A]
My camera is mounted HI up in the air.  Any way to update the firmware remotely.  I really don't want to get a later out to pop it out and bring it down for a card swap.
(i have remote ssh access)

Thanks in advanced.

**Comment by gtxaspec** [comment]
upgrade-run.sh from the command line

**Comment by virmaior** [comment]
if it's an older version before upgrade-run.sh was a part of the package, then you can do it using upgrade-run.sh  .  I've outlined how at https://github.com/gtxaspec/wz_mini_hacks/discussions/117 . I won't guarantee that works, but it worked for a camera for me that is a pain to move.
